src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -146,10 +149,48 @@ public void Verify()
             _instructionBoundaries = new bool[_ilBytes.Length];
 
             FindBasicBlocks();
-
+            FindEnclosingExceptionRegions();
             ImportBasicBlocks();
         }
 
+        private void FindEnclosingExceptionRegions()
+        {
+            for (int i = 0; i < _basicBlocks.Length; i++)
+            {
+                if (_basicBlocks[i] == null)
+                    continue;
+
+                var basicBlock = _basicBlocks[i];
+                var offset = basicBlock.StartOffset;
+
+                //find enclosing try
+                for (int j = 0; j < _exceptionRegions.Length; j++)
```

Comment:
There is faster algorithm to do this, but this one is fine at least for now. Maybe nice to add TODO command about it